datalad/support/gitrepo.py

```diff
@@ -697,7 +710,7 @@ def __eq__(self, obj):
     @classmethod
     def is_valid_repo(cls, path):
         """Returns if a given path points to a git repository"""
-        return exists(opj(path, '.git', 'objects'))
+        return exists(opj(path, '.git'))
 
```

Comment:
There is no certainty, of course, but the likelihood is different -- more bits in the tested information ;-)

Anyways, good enough.